services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -659,7 +662,27 @@ module.exports = {
         maxResults,
         ...(pageToken && { pageToken }),
       };
-      return googleDataApiClient.request('youtube.playlistItems', payload);
+      const response = await googleDataApiClient.request('youtube.playlistItems', payload);
+      const find400 = [];
+      if (getAsArray(response, 'errors')) { // If the response contains errors
+        const errors = getAsArray(response, 'errors');
+        errors.forEach((error) => {
+          if (error.extensions.exception.statusCode >= 400) { // If there is a 400 error
+            find400.push(error);
+          }
+        });
+      }
+      if (find400.length > 0) {
+        return { pageInfo: {}, items: [] }; // If a 400 error is found return "nothing"
+      }
```

Comment:
These errors should be returned to the client to handle. Further, the pageInfo and items should likely be returned as-is as well